MusicNotationKit/MusicNotationKit/Clef.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  Clef.swift
+//  MusicNotationKit
+//
+//  Created by Kyle Sherman on 10/16/16.
+//  Copyright © 2016 Kyle Sherman. All rights reserved.
+//
+
+public struct Clef {
+
+    public let tone: Tone?
+    /**
+     Starts from 0 on the first line (from the bottom). Ledger lines below that are negative.
+     Each increase by 1 moves a half step. i.e. 1 is the first space on the staff.
+     */
+    public let lineNumber: Int
```

Comment:
Yeah although we would still represent ledger lines (and associated spaces) this way as well. So it would really be negative to negative infinity and positive infinity for lines and spaces, right?